scripts/col-repl.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright ©2016. The Regents of the University of California (Regents). All Rights Reserved.
+ *
+ * Permission to use, copy, modify, and distribute this software and its documentation
+ * for educational, research, and not-for-profit purposes, without fee and without a
+ * signed licensing agreement, is hereby granted, provided that the above copyright
+ * notice, this paragraph and the following two paragraphs appear in all copies,
+ * modifications, and distributions.
+ *
+ * Contact The Office of Technology Licensing, UC Berkeley, 2150 Shattuck Avenue,
+ * Suite 510, Berkeley, CA 94720-1620, (510) 643-7201, otl@berkeley.edu,
+ * http://ipira.berkeley.edu/industry-info for commercial licensing opportunities.
+ *
+ * IN NO EVENT SHALL REGENTS BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+ * INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF
+ * THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF REGENTS HAS BEEN ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * REGENTS SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
+ * SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, PROVIDED HEREUNDER IS PROVIDED
+ * "AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ * ENHANCEMENTS, OR MODIFICATIONS.
+ */
+
+var _ = require('lodash');
+var DB = require('col-core/lib/db');
+
+/**
+ * Load Collabosphere environment and start REPL
+ */
+console.log('Loading Collabosphere environment...');
+
+var init = function() {
+  // Apply global utilities
+  require('col-core/lib/globals');
+    
+  // Connect to the Collabosphere database
+  DB.init(function() {
+
+    // Require additional modules and utilities, assigning them to properties 
+    // to be exported to the REPL.
+
+    var exports = {
+      // External modules
+      'async': require('async'),
+      'config': require('config'),
+      'Joi': require('joi'),
+      // Node's REPL treats _ as a special variable, returning the result of the last 
+      // expression, so we can't assign to it.
+      'lodash': _,
+      'moment': require('moment-timezone'),
+      'Sequelize': require('sequelize'),
+
+      // Collabosphere modules
+      'ActivitiesAPI': require('col-activities'),
```

Comment:
Same here. How do we decide what to include (e.g., the Whiteboards API is not included)